### PR TITLE
Server input

### PR DESCRIPTION
**Pull request by luwol03**
| Status  | Type  |
| :---: | :---: |
|  :white_check_mark: Ready | Feature |

## TODO

- [x] check server version on login
- [x] extract origin from URL in server input field
- [x] show server version on settings page 
- [x] improve style (maybe @zikowang can help me)

## Checklist

- [x] I have read the **CONTRIBUTING** document.
- [x] I have considered the accessibility of my changes (i.e. did I add proper content descriptions to images, or run my changes with talkback enabled?)
- [x] I have documented my code if needed

### CODE REVIEW

File: src/Components/Indicators/ServerValidIndicator/ServerValidIndicator.jsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ const ServerValidIndicator = ({ setValid }) => {
         }
       })
       .finally(() => {
-        setIsLoading(false);
+        setTimeout(() => setIsLoading(false), 500);
```
Comment (luwol03) [inline src/Components/Indicators/ServerValidIndicator/ServerValidIndicator.jsx:48]: what is the purpose behind the timeout? If we want to leave this, we still have to put it into the `useEffect` cleanup function.

File: src/screens/Login/Login.jsx [hunk 2]
```diff
@@ -145,9 +145,10 @@ const Login = ({ image }) => {
                 value={serverAddressInput}
               />
             )}
-            <ServerValidIndicator setValid={setIsServerValid} />
-            {serverError && (
+            {serverError ? (
```
Comment (luwol03) [inline src/screens/Login/Login.jsx:148]: shouldn't we improve this also in Register.jsx?

**Review by luwol03 — COMMENTED** [review]
Could you eventually still look over the authendicated layout?
image
That doesn't look so good yet.

File: src/Components/Indicators/ServerValidIndicator/ServerValidIndicator.jsx (hunk 1 above)
Comment (zikowang) [reply src/Components/Indicators/ServerValidIndicator/ServerValidIndicator.jsx]: The loading spinner and status strings are jumping without a timeout.

I've cleaned it. Please take a look.

File: src/screens/Login/Login.jsx (hunk 2 above)
Comment (zikowang) [reply src/screens/Login/Login.jsx:148]: Yes sir.

**Comment by zikowang** [comment]
> Could you eventually still look over the authendicated layout?
> That doesn't look so good yet.

Done

**Review by zikowang — APPROVED** [review]

**Pull request merged**